lib_agc/src/agc.xc

```diff
@@ -576,17 +576,13 @@ static void agc_process_channel(agc_ch_state_t &state, vtb_ch_pair_t samples[AGC
 
         // Update near-end-activity timer
         if(vtb_gte_u32_u32(state.lc_near_power_est, vtb_mul_u32_u32(delta, state.lc_bg_power_est))){
-            if(state.lc_t_far == 0){
-                // Near-end speech only
+            if(state.lc_t_far == 0 || (state.lc_t_far && (state.lc_corr_factor < state.lc_corr_threshold.m))){
```

Comment:
Is the `lc_t_far` element of the `state` variable a Boolean or not?  If it is a Boolean, then use only Boolean operators, i.e., `!state.lc_t_far` instead of `state.lc_t_far == 0`.  If it isn't a Boolean, then do not use Boolean operators, i.e., `state.lc_t_far != 0` instead of `state_lc_t_far`.

In this file, the `lc_t_far` element is used sometimes as a Boolean and sometimes not.  I should be used consistently throughout.

Reply:
I included the review comments in the commit:
https://github.com/xmos/lib_agc/pull/150/commits/af609b739d5452f3a1b1c7ae921237f797848492